alembic/versions/d05256bc14ed_automatically_publish_source_when_.py

```diff
@@ -0,0 +1,37 @@
+"""Automatically publish source when created
+
+Revision ID: d05256bc14ed
+Revises: 3eb663b5c3e0
+Create Date: 2025-01-15 16:55:03.008848
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd05256bc14ed'
+down_revision = '3eb663b5c3e0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        'publicreleases',
```

Comment:
weird that the 2 columns have almost the same name but not really. Please make sure that they are identical if they mean the same thing

Reply:
this comment applies to all the files that use these columns